packages/unlock-express/src/keyExpirationFor.js

```diff
@@ -0,0 +1,61 @@
+const https = require('https')
+/**
+ * Wrapper around keyExpirationFor
+ * @param {*} provider
+ * @param {*} lock
+ * @param {*} userAddress
+ * @returns
+ */
+async function keyExpirationFor(provider, lock, userAddress) {
+  return new Promise((resolve, reject) => {
+    const data = JSON.stringify({
```

Comment:
This is some elite web3 super coder magic

Reply:
oh yeah :) Devs should not have to worry about this tho.